docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM php:5-apache
+
+RUN a2enmod rewrite
+RUN apt-get update && \
+    apt-get install -y libxml2 libxml2-dev && \
+    docker-php-ext-install mysqli soap dom
+RUN yes | pecl install xdebug \
+    && echo "zend_extension=$(find /usr/local/lib/php/extensions/ -name xdebug.so)" > /usr/local/etc/php/conf.d/xdebug.ini \
+    && echo "xdebug.remote_connect_back=0" >> /usr/local/etc/php/conf.d/xdebug.ini
+    && echo "xdebug.remote_host=docker.for.mac.host.internal" >> /usr/local/etc/php/conf.d/xdebug.ini
```

Comment:
I wonder if this should be left out as well. It is very specific to the Docker for mac use case.